### DISCUSSION TITLE
How to add jekyll SEO plugin to chirpy starter

### DISCUSSION THREAD

**Discussion by qureshiaquib** [cotes2020/jekyll-theme-chirpy#1652] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### What is the topic?

Sharing tips and tricks

### Description

I've forked a chirpy starter repo and deployed it on github pages.
I've found about jekyll seo plugin and wanted to understand can I now manually install plugin and where should I add %seo%? Or Is this now builtin and I should not worry about this? 

My site has got lot of content which is unique and it been more than 3 months but my site is not being indexed properly via Google. 

Will there be any impact to my blog if I do jekyll seo plugin now? And any guide would help. 

Thanks in advance

Site for reference : www.azuredoctor.com

**Comment by kungfux** [comment]
SEO settings can be configured in [_config.yml](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/_config.yml#L14).

We already had a [discussion](https://github.com/cotes2020/jekyll-theme-chirpy/discussions/1514) about being indexed. Please check it out.

**Comment by qureshiaquib** [comment]
Thanks let me check this out.

**Reply by kungfux** [reply, marked as answer]
> Thanks kungfux, I thought this seo tag is suppose to be there in every post.but it seems we've single place to configure this.
Is this through jekyll seo plugin?

Well, you configure it once and SEO tags will produce for every post.

You can check `<head>` section of Chirpy [post](https://chirpy.cotes.page/posts/getting-started/) for example.

**Comment by qureshiaquib** [comment]
Hi Alex,

This will automatically get indexed once you put all the SEO related information in cofig.yaml file.
The title, google verification,  descriptions author information was very important as these get tagged in every post.

Regards,

Aquib Qureshi

________________________________
